docs/guides/avoid_blocking.md

```diff
@@ -0,0 +1,49 @@
+---
+id: avoid-blocking-fingerprints
+title: Avoid getting blocked
+---
+TODO: this is only for browser-based scraping. We should reflect in the name somehow... IMHO, browser fingerprinting does not remember that it reduces blocking.
+
+
+Your scraper might get blocked for numerous reasons. Let's narrow it down to two main reasons. The first one is a bad or blocked IP address. This topic is covered in the [proxy management guide](proxy_management.MD). The second reason we will explore more is browser fingerprints or signatures.
```

Comment:
Better to use `we/our` then `you/your`, or even just `A scraper might` or `Scrapers might`.

also the link is wrong, `.MD` vs `.md`